### ISSUE TITLE
Country names not in line with openentrance definitions

### ISSUE THREAD

**Issue by HauHe** [OSeMOSYS/osemosys2iamc#38]
When trying to upload results to the IIASA ScenarioExplorer I noticed that the recent fixes on country names are not in line with the definitions in the `openentrance` package.

| We report  | OpenEntrance requires  |
|---|---|
|  Netherlands | The Netherlands  |
| Czechia  | Czech Republic  |
| United Kingdom of Great Britain and Northern Ireland  | United Kingdom  |

**Comment by danielhuppmann** [comment]
You are right, "Czechia" is a relatively new change, the other two items are very common problems.

And yes, in openENTRANCE, we use an older list of country names, which we are currently trying to standardize across projects at https://nomenclature-iamc.readthedocs.io/en/stable/api/countries.html.

My suggestion is that you implement all reporting using the nomenclature country names (Netherlands, Czechia, United Kingdom) and I add a "manual" translation in the openENTRANCE workflow to ensure compatibility in that project. Sounds ok?

**Comment by HauHe** [comment]
Hi Daniel,
yes, we will create a fix so that we report with the country names used in openEntrance, i.e. the Netherlands, Czech Republic etc.

**Comment by willu47** [comment]
Closed by #40
